src/main/connector/system/Windows/Windows.yaml

```diff
@@ -204,19 +204,49 @@ monitors:
               BytesSentPerSec,
               BytesReceivedPerSec
             FROM Win32_PerfRawData_Tcpip_NetworkInterface
+          computes:
+            - type: duplicateColumn
+              column: 1
+            - type: awk
+              script: 'BEGIN {FS=OFS=";"} {gsub(/[^a-zA-Z0-9]/, " ", $2); print $0}'
+        # Name;IpAddress
+        networkAdapters:
+          type: wmi
+          namespace: root\CIMv2
+          query: >
+            SELECT Description,
+              IPAddress
+            FROM Win32_NetworkAdapterConfiguration
+          computes:
+            - type: duplicateColumn
+              column: 1
+            - type: awk
+              script: 'BEGIN {FS=OFS=";"} {gsub(/[^a-zA-Z0-9]/, " ", $2); print $0}'
+            - type: duplicateColumn
+              column: 3
+            - type: awk
+              script: 'BEGIN {FS=OFS=";"} {gsub(/\|/, " ", $4); print $0}'
+        networkInformation:
+          type: tableJoin
+          leftTable: ${source::networkAdapters}
+          rightTable: ${source::networkInterfacePrefs}
+          leftKeyColumn: 2
+          rightKeyColumn: 2
+          defaultRightLine: ;;;;;;;;
       mapping:
         source: ${source::networkInformation}
         attributes:
-          id: $1
+          id: $2
+          ip: $4

```

Comment:
In Otel semantic conventions, the IP addresses is to be represented with the `network.local.address` attribute.

See: https://opentelemetry.io/docs/specs/semconv/attributes-registry/network/